### DISCUSSION TITLE
Does Stats API not work in the offseason

### DISCUSSION THREAD

**Discussion by DomZ31** [MLB-LED-Scoreboard/mlb-led-scoreboard#506] [Q&A]
I have set this up before during baseball season but had hardware issues, fixed the hardware issues and now it says stats API is down is that just because its the offseason?

**Comment by ty-porter** [comment]
StatsAPI is not down to my knowledge -- it could have been temporarily perhaps, but there's not much we can do about it as that is being provided by MLB.

Otherwise it sounds like a network issue. Without an actual error message it is impossible to tell.

**Reply by DomZ31** [reply]
Thanks Ty, should I move this over to the issues section then. The messages I am getting are "No module named 'statsapi', my guess is that there is also 4 other files that did not download properly.

**Reply by ty-porter** [reply]
This doesn't really sound like an issue with the software. Make sure you're running the installer to grab the latest updates and dependencies:

```sh
sudo ./install.sh
```

If you run into more errors you can post them here (please copy and paste or take a screenshot, paraphrasing is not sufficient).

If it turns out to be a widespread issue we can move it out of discussions.

**Comment by twfarley** [comment]
The installation should take care of that. Make sure you’re running that. If you see any errors there be sure to mention them here. It can’t hurt to run it again. On Nov 27, 2023, at 8:11 PM, DomZ31 ***@***.***> wrote:﻿
Thanks Ty, should I move this over to the issues section then. The messages I am getting are "No module named 'statsapi', my guess is that there is also 4 other files that did not download properly.

**Reply by DomZ31** [reply]
I don't know what I did different but I tried deleting and reinstalling everything multiple times and had no luck, I finally updated rasp imager and tried again and seem to have gotten everything up and running just need to make some small personal tweaks but should be good. Thanks for all the help and I can't wait to finish the 3d prints and get this mounted.
